docs/index.md

```diff
@@ -509,6 +509,10 @@ Its most distinctive features are built-in support for dependency injection, aut
 
 [Falcon](https://falconframework.org) is a minimalist REST and app backend framework for Python, with a focus on reliability, correctness, and performance at scale.
 
+### Muffin
+
+[Muffin](https://github.com/klen/muffin) is a fast, lightweight and asyncronous ASGI web-framework for Python 3.
```

Comment:
Is asyncronous a typo or this spelling exists?

Reply:
sorry, must be a typo